baggage/HTTP_HEADER_FORMAT_RATIONALE.md

```diff
@@ -5,7 +5,7 @@ This document provides a rationale for the decisions made for the `baggage` head
 ## General considerations
 
 - It should be human-readable. Cryptic headers would hide the fact of potential information disclosure.
-- It should be append-able (comma-separated) https://tools.ietf.org/html/rfc7230#page-24 so nodes
+- It should be appendable (comma-separated) https://tools.ietf.org/html/rfc7230#page-24 so nodes
```

Comment:
```suggestion
- It should be appendable (comma-separated) https://tools.ietf.org/html/rfc7230#page-24, so nodes
```

Reply:
Thanks, will do this in a separate PR to make editorial edits on the overall file.